modules/340-extended-monitoring/images/image-availability-exporter/Dockerfile

```diff
@@ -1,11 +1,11 @@
 ARG BASE_ALPINE
-ARG BASE_GOLANG_17_ALPINE
+ARG BASE_GOLANG_19_ALPINE
 
 # Based on https://github.com/deckhouse/k8s-image-availability-exporter/blob/master/Dockerfile
-FROM $BASE_GOLANG_17_ALPINE as artifact
+FROM $BASE_GOLANG_19_ALPINE as artifact
 WORKDIR /src
 ENV GOARCH=amd64
-RUN wget https://github.com/deckhouse/k8s-image-availability-exporter/tarball/v0.3.4 -O - | tar -xz --strip-components=1 && \
+RUN wget https://github.com/deckhouse/k8s-image-availability-exporter/tarball/44f18a343c3266be366faea4905ef91093aa0039 -O - | tar -xz --strip-components=1 && \
```

Comment:
Which version is that? How we can easily find appropriate changelogs to find out we should updgrade it?

Reply:
Released and changed `v0.4.0`.